src/spikeinterface/widgets/traces.py

```diff
@@ -203,6 +217,8 @@ def __init__(
             else:
                 raise TypeError(f"'clim' can be None, tuple, or dict! Unsupported type {type(clim)}")
 
+        self.cb = None
```

Comment:
To support multiple backend we have to avoid self in the init.
Put this in the data_plot.